CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ ENHANCEMENTS:
 BUG FIXES:
 * Letsencrypt.yml fails with "Invalid reference in variable validation" ([#4506](https://github.com/microsoft/AzureTRE/4506))
 * Intermittent management storage account access failure during core deployment ([#4505](https://github.com/microsoft/AzureTRE/4505))
+* Fix AirLock VM not deleting ([#4474](https://github.com/microsoft/AzureTRE/issues/4474))
```

Comment:
What makes an Airlock VM more prone to this issue vs a regular VM?

Reply:
It has two nics.